programs/cw-ics08-wasm-eth/src/query.rs

```diff
@@ -54,8 +56,10 @@ pub fn timestamp_at_height(
     let eth_consensus_state =
         get_eth_consensus_state(deps.storage, timestamp_at_height_msg.height.revision_height)?;
 
+    let nano_timestamp = eth_consensus_state.timestamp.mul(1_000_000_000); // ibc-go expects nanoseconds
```

Comment:
why do we use `.mul` here but `* 1_000_000_000` elsewhere. I think we should stick with `* 1_000_000_000` for consistency

Reply:
I've addressed this. Lmk if you see an issue